NavCog3.xcodeproj/project.pbxproj

```diff
@@ -467,6 +467,22 @@
 		A9ED1549213914A6009E1E0C /* ko */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = ko; path = SettingUtil/ko.lproj/HLPSettingView.strings; sourceTree = "<group>"; };
 		ABCC51EA52F7E70CF93DA9D2 /* libPods-NavCog3.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; includeInIndex = 0; path = "libPods-NavCog3.a"; sourceTree = BUILT_PRODUCTS_DIR; };
 		D5D9DE6372649AF633976E29 /* libPods-NavCogFP.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; includeInIndex = 0; path = "libPods-NavCogFP.a"; sourceTree = BUILT_PRODUCTS_DIR; };
+		E201FC5E2875E17700A9ADDD /* libPods-NavCogTool.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; path = "libPods-NavCogTool.a"; sourceTree = BUILT_PRODUCTS_DIR; };
+		E201FC602875E81100A9ADDD /* libPods-NavCogTool.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; path = "libPods-NavCogTool.a"; sourceTree = BUILT_PRODUCTS_DIR; };
+		E201FC622875E82600A9ADDD /* libPods-NavCogTool.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; path = "libPods-NavCogTool.a"; sourceTree = BUILT_PRODUCTS_DIR; };
+		E201FC77287863C500A9ADDD /* libPods-NavCogTool.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; path = "libPods-NavCogTool.a"; sourceTree = BUILT_PRODUCTS_DIR; };
+		E201FC79287863D900A9ADDD /* libPods-NavCogTool.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; path = "libPods-NavCogTool.a"; sourceTree = BUILT_PRODUCTS_DIR; };
+		E2082D1E28B4B9F2008492D1 /* ar */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = ar; path = ar.lproj/Fingerprint.strings; sourceTree = "<group>"; };
```

Comment:
You may need to commit `ar.lproj/Fingerprint.strings` which is not included in this PR.

Reply:
done , but the file Is empty .